htdocs/scss/pages/entry/new.scss

```diff
@@ -75,6 +75,7 @@
 
     li {
         display: inline-block;
+        margin-left: 0 !important;
```

Comment:
It's used for things that should never be overridden by more specific targets, so it's appropriate here.  There are other lines in the same SCSS file that also use it.